compiler-rt/test/builtins/Unit/floattitf_test.c

```diff
@@ -7,7 +7,7 @@
 #include <float.h>
 #include <stdio.h>
 
-#if defined(CRT_HAS_TF_MODE)
+#if defined(CRT_HAS_TF_MODE) && defined(CRT_LDBL_IEEE_F128)
```

Comment:
floattitf is being compiled when CRT_HAS_TF_MODE. Why are we additionally guarding this test with CRT_LDBL_IEEE_F128

Reply:
Again most of the results assume IEEE format, so they fail for PPC64 with IBM format.

Reply:
Added the correct check.